### ISSUE TITLE
Wrong method name

### ISSUE THREAD

**Issue by sokolandrei** [mopub/mopub-unity-sdk#16]
https://github.com/mopub/mopub-unity-sdk/blob/79b7fa12f09e2ebb14b35c80e56611f739141ec4/mopub-android-sdk-unity/src/main/java/com/mopub/unity/MoPubRewardedVideoUnityPlugin.java#L226

Should be `RewardedVideoReceivedReward`, not `RewardedVideoReceived` to receive event in Unity: https://github.com/mopub/mopub-unity-sdk/blob/79b7fa12f09e2ebb14b35c80e56611f739141ec4/unity-sample-app/Assets/MoPub/MoPubManager.cs#L369

**Comment by aornelas** [comment]
Thanks for your report! We'll include this fix on the next release.

Cheers!

**Comment by mertkurum** [comment]
There is also one more naming issue on IsConsentDialogLoaded. It is used as IsConsentDialogReady on Android.

**Comment by HugoLec** [comment]
A little fix in waiting of newt release : 

Add this method in the `MoPubManager.cs` (unity Side, class imported by the unitypackage)

```
    public void EmitRewardedVideoReceivedEvent(string argsJson)
    {
        var args = DecodeArgs(argsJson, min: 3);
        var adUnitId = args[0];
        var label = args[1];
        var amountStr = args[2];
        
        var evt = OnRewardedVideoReceivedRewardEvent;
        if (evt != null) evt(adUnitId, label, float.Parse(amountStr));
    }
```

This is the same implemetation of method `EmitRewardedVideoReceivedRewardEvent` with the good signature. And the two platform `Android`and `iOs` is supported

Thanks for the issue

**Comment by aornelas** [comment]
This has been fixed in the [5.1.0 release](https://github.com/mopub/mopub-unity-sdk/releases/tag/v5.1.0).

Thanks again for reporting!
